packages/web-client/tests/acceptance/not-found-test.ts

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | not-found', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('it shows a 404 page for unknown routes', async function (assert) {
+    await visit('/blah');
+
+    assert.equal(currentURL(), '/blah');
+
+    assert
+      .dom('[data-test-not-found-page]')
+      .containsText('The page you are looking for does not exist');
+
+    assert.dom('[data-test-back-home-link]').exists().hasAttribute('href', '/');
+  });
+});
```

Comment:
Percy perhaps? 🤓

Reply:
Ya I wondered about that but was unsure how much value it brings, esp since our usage is so high